src/library/basetracktablemodel.cpp

```diff
@@ -625,7 +625,11 @@ QVariant BaseTrackTableModel::roleValue(
             VERIFY_OR_DEBUG_ASSERT(rawValue.canConvert<QDateTime>()) {
                 return QVariant();
             }
-            return mixxx::localDateTimeFromUtc(rawValue.toDateTime());
+            QDateTime dt = mixxx::localDateTimeFromUtc(rawValue.toDateTime());
```

Comment:
You need to enclose the case block in a scope now, like the next one. C++ weirdness.

Reply:
ah yes of course